tools.go

```diff
@@ -0,0 +1,17 @@
+// +build tools
+
+// tools is a dummy package that will be ignored for builds, but included for dependencies.
+package tools
+
+import (
+	// Code generators built at runtime.
+	_ "k8s.io/code-generator" // TODO: Investigate why scripts in this directory are removed and not vendored by go mod.
+	_ "k8s.io/code-generator/cmd/client-gen"
+	_ "k8s.io/code-generator/cmd/conversion-gen"
+	_ "k8s.io/code-generator/cmd/deepcopy-gen"
+	_ "k8s.io/code-generator/cmd/defaulter-gen"
+	_ "k8s.io/code-generator/cmd/informer-gen"
+	_ "k8s.io/code-generator/cmd/lister-gen"
```

Comment:
we can move go-bindata and hopefully golangci-lint here as suggested in https://github.com/openshift/machine-config-operator/pull/961#issuecomment-511134245 (but this is a comment for a follow up I'm experimenting with)